code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -295,3 +295,12 @@
 			user << "Nearest operative detected is <i>[nearest_op.real_name].</i>"
 		else
 			user << "No operatives detected within scanning range."
+			
+/obj/item/weapon/pinpointer/wizard
+	name = "wiz pointer"
+	desc = "Tracks wizards with extreme precision; down the exact locker they are hiding in."
+		
+/obj/item/weapon/pinpointer/attack_self()
+	for(var/datum/mind/wizard in ticker.mode.wizards)

```

Comment:
Going to test it in a few hours and find out! If this doesn't work I think get_turf for the mind should.